code/modules/posture/posture_bodytype.dm

```diff
@@ -0,0 +1,26 @@
+/decl/bodytype
+	// This will generally not be used, except when a posture is not present in available_mob_postures.
+	var/list/basic_posture_map = list(
```

Comment:
what exactly is this? is this a list of fallbacks or something? why's it necessary, is this ever set? changing available postures without changing this would still result in invalid postures from `get_equivalent_posture_type` i think

Reply:
ah alright, makes sense. though maybe in the future we can revisit it and have it use a key on the posture decl